linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -122,7 +122,7 @@ private void run(String groupId, String artifactId)
         for (ClassFile classFile : snapshotSymbolProblems.get(problem)) {
           message.append(
               String.format(
-                  "  referenced from %s (%s)",
+                  "  referenced from %s (%s)\n",
```

Comment:
This method has gotten too big and complex to easilly follow. Plus it looks like it's undertested siunce this change didn't b reak anything. We should consider breaking it up.

Reply:
Created #742